controllers/suggestions/suggestions_review_base_controller.py

```diff
@@ -1,11 +1,17 @@
-from consts.account_permissions import AccountPermissions
-
 from controllers.base_controller import LoggedInHandler
 
+
 class SuggestionsReviewBaseController(LoggedInHandler):
     """
     Base controller for reviewing suggestions.
     """
+
+    REQUIRED_PERMISSIONS = []
+
     def __init__(self, *args, **kw):
         super(SuggestionsReviewBaseController, self).__init__(*args, **kw)
-        self._require_permission(AccountPermissions.MUTATE_DATA)
+        self.verify_permissions()
+
+    def verify_permissions(self):
+        for permission in self.REQUIRED_PERMISSIONS:
```

Comment:
Greg wrote the original class though, I just tweaked it. Anyway, I think it's possible that there will be more than one required permissions for a page, so it's better to keep as a list